conda-forge.yml

```diff
@@ -1,6 +1,6 @@
 travis:
   secure:
-    BINSTAR_TOKEN: Wq9fYw4rhyXMGPYoGiHG+f7iYmPe+SJPoJ4ctaz2YGMb1SWihaKJ7iVvOey9t8GHx9q5bKPAtP+4sBmaPZ4L8GWVwyfGcQEjri30W2BwCqoaQcVUo9U9FXUyblDcJkqTCtJnuxjUFz0Uo7CE1pmj1+BoSwVnfPq6LIJ4oBD03xtz3X/PRO5rzCgB0FOCKPnJBK3FtOhhU9ezQPISDnQ7JbsNYU9x561WsJYub/qdaHVwtTrNi/kxgru0R/bJIPJ8TfagmspcdsPmrOHn1UU3UgU/6UEK3pyEF5cI+X/sWWFzahuK8skInAu2djbwq9+rxzH9YsoV5KtxSkFlOQOZ0Y09yGOXNZqNQUBDGIX7lcGtXjZgtGW39Qx/yX35kllJnwiEOGInVDOPwuHNBwRjPkFopURsqFNwhteDjdegbJM/bb3i5uJBIl/Va037S+chkwEmo2DYHNmb2M9q97tRgcxEVDcmlR8EVoOfpeJRXm4dnOIoFVpYjJiHXL2N55Va04eBHJvltXlGv2lkowYiIeDGgY7ciTisznbOfRNk5mbIS9caMgENRswXuuz1Nc5HRzvw5yIyfY9aBy6Huxa8FHeku7NgiQHf8+vETj+DL6DQb856vzxXqjw+EUYkXWPCU+R7NbBc4YX8Pax4yOzgLvBuH8QqT8t+VnBP0nl8WRg=
+    BINSTAR_TOKEN: SCGkF4vEzClVTXZpj8qaElCOoIYasXSD642mHFHHBPZS9H9TorVrUd0o1EwFL1aCxrwrVOO41yhiGkLCu6OyO32yPyv2b644FEyNsjKRoQbbjoA2u/gGdj4fP95Pr4xgqU4LQOqcHXDcG1a/hejZc1bCTLwCm2r8exF8GbAGlp6eQ45NYq6RjekeS7rMh6IW8wRxs5RR67Nowa+eiBIOwLI7pj0GPyx2XEi0R5YAik69AiYNbBP7hXA3913PRQpLmoEZknUvUAzAnQ6/1v4SmJNjnXjHwtmy5spqdWneaD0RA+UqEhHMKzDn5YZu9FT17lw5FHuMLgHcFDTIMGUMEjiKjRGTBbG9NnlgEpJAGSzal4U0A9gdfYKpDwKLKZ2JaxXcwAYAeZ48An74KQceRDG8lYb+olrQthUhQDbFZU80VIiQMQukAtQe4OYYtVtj9iAcAqHjsz/EoxJIeTbabSsMJsDTJG7+9vM3fbtLCSQ06v4kLSzlX5l9xTicLK/TmM8AQIVnBhQmxNAKq2uW/Ehmb3tkcWGeg5G8GUXrD8UJabYtXTm0B+LFiCbZmkw+a5+a/D/0P7R4Amt/ql+JaHwCJ+vHW9+TJHu5LptSgWutD18HOIZkl7VVRLpw8ZWa97mYtDnr1DpRvvopmdNS73T8Ssdyb7Pesky0SuseIwA=
```

Comment:
This change is what is causing https://github.com/conda-forge/conda-smithy/issues/504